### ISSUE TITLE
NTR: Lauric acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8631]
New term request Lauric acid high in urine (segal_070726175025): Dodecanoic

**Comment by pnrobinson** [comment]
@MickeySegal
is this a xenobiotic? What is the context?

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000434
- label: Elevated urinary isovaleric acid level
- parents: [Organic aciduria (HP:0001992)](https://hpo.jax.org/app/browse/term/HP:0001992)
- pmid: 
- def: The amount of isovaleric acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8632
- synonyms:

**Comment by MickeySegal** [comment]
Lauric acid is very different from isovaleric acid.  There is a commercial test at https://neurology.testcatalog.org/show/FAPM and structure at https://en.wikipedia.org/wiki/Lauric_acid

We have it for glutaric aciduria type II referenced to an old textbook, but it is not mentioned at https://omim.org/entry/231680 so it may be no longer tested.

**Comment by MickeySegal** [comment]
I now get web ads from Sigma-Aldrich selling isovaleric acid.
